src/reynir/matcher.py

```diff
@@ -228,66 +310,106 @@
     "NP-POSS": dict(name="Eignarfallsliður", overrides="NP"),
     "NP-DAT": dict(name="Þágufallsliður", overrides="NP"),
     "NP-ADDR": dict(name="Heimilisfang", overrides="NP"),
     "NP-COMPANY": dict(name="Fyrirtæki", overrides="NP"),
     "NP-TITLE": dict(name="Titill", overrides="NP"),
     "NP-AGE": dict(name="Aldur"),
-    "NP-MEASURE": dict(name="Mæling", overrides="NP"),
+    "NP-MEASURE": dict(name="Magnliður", overrides="NP"),
+    #"NP-PERSON": dict(name="Manneskja"),
     "NP-SUBJ": dict(name="Frumlag", subject_to={"NP-SUBJ"}),
     "NP-OBJ": dict(name="Beint andlag"),
     "NP-IOBJ": dict(name="Óbeint andlag"),
     "NP-PRD": dict(name="Sagnfylling"),
+    "NP-ADP": dict(name="Andlag lýsingarorðs"),
     "ADVP": dict(name="Atviksliður", subject_to={"ADVP"}),
-    "ADVP-DATE": dict(name="Dagsetning"),
-    "ADVP-DATE-ABS": dict(name="Föst dagsetning"),
-    "ADVP-DATE-REL": dict(name="Afstæð dagsetning"),
-    "ADVP-TIMESTAMP": dict(name="Tímapunktur"),
-    "ADVP-TIMESTAMP-ABS": dict(name="Fastur tímapunktur"),
-    "ADVP-TIMESTAMP-REL": dict(name="Afstæður tímapunktur"),
-    "ADVP-TMP-SET": dict(name="Tíðni"),
-    "ADVP-DUR": dict(name="Tímabil"),
-    "ADVP-DUR-ABS": dict(name="Fast tímabil", overrides="ADVP-DUR"),
-    "ADVP-DUR-REL": dict(name="Afstætt tímabil", overrides="ADVP-DUR"),
-    "ADVP-DUR-TIME": dict(name="Tímabil", overrides="ADVP-DUR"),
-    "PP": dict(name="Forsetningarliður", overrides="ADVP"),
-    "ADJP": dict(name="Lýsingarliður", subject_to={"ADJP"}),
-    "IP": dict(name="Beygingarliður"),  # Inflectional phrase
+    "ADVP-DIR": dict(name="Áttaratviksliður"),
+    "ADVP-DATE-ABS": dict(name="Föst dagsetning", overrides="ADVP"),
+    "ADVP-DATE-REL": dict(name="Afstæð dagsetning", overrides="ADVP"),
+    #"ADVP-TIMESTAMP": dict(name="Tímapunktur", overrides="ADVP"),
+    "ADVP-TIMESTAMP-ABS": dict(name="Fastur tímapunktur", overrides="ADVP"),
+    "ADVP-TIMESTAMP-REL": dict(name="Afstæður tímapunktur", overrides="ADVP"),
+    "ADVP-TMP-SET": dict(name="Tíðni", overrides="ADVP"),   
+    "ADVP-DUR-ABS": dict(name="Fast tímabil"),
+    "ADVP-DUR-REL": dict(name="Afstætt tímabil", overrides="ADVP"),
+    "ADVP-DUR-TIME": dict(name="Tímabil"),
+    "ADVP-PCL": dict(name="Ögn"),
+    "PP": dict(name="Forsetningarliður", overrides="ADVP", subject_to={"ADVP-DUR-REL", "ADVP-DUR-ABS"}),
+    #"ADJP": dict(name="Lýsingarliður", subject_to={"ADJP"}),
+    # Hausar
+    #"ADV": dict(name="Atviksorð"),
+    #"V": dict(name="Sögn"),
+    #"N": dict(name="Nafnorð"),
+    #"PRON": dict(name="Fornafn"),
+    "P": dict(name="Forsetning"),
+    "TO": dict(name="Nafnháttarmerki"),
+    #"NUM": dict(name="Töluorð", subject_to={"NUM-RANGE"}),
+    "C": dict(name="Samtenging"),
+    #"ADJ": dict(name="Lýsingarorð", overrides="V"),
+    #"DET": dict(name="Greinir"),
+    #"DATEREL": dict(name="Afstæð dagsetning"),
+    #"DATEABS": dict(name="Föst dagsetning")
+    "FOREIGN": dict(name="Erlendur texti"),
```

Comment:
Should we use `S0-FOREIGN` for sentences that are probably foreign?

Reply:
The idea was to let `S0-X `cover all sentences with errors or foreign text but it might make sense to separate them, although it might not always be possible.